workers/go/kitchensink/kitchen_sink.go

```diff
@@ -152,8 +196,45 @@ func launchActivity(ctx workflow.Context, act *kitchensink.ExecuteActivityAction
 			RetryPolicy:            convertFromPBRetryPolicy(act.GetRetryPolicy()),
 		}
 		actCtx := workflow.WithActivityOptions(ctx, opts)
-		return workflow.ExecuteActivity(actCtx, act.ActivityType, act.Arguments).Get(actCtx, nil)
+		return withAwaitableChoice(actCtx, func(ctx workflow.Context) workflow.Future {
+			return workflow.ExecuteActivity(ctx, act.ActivityType, args)
+		}, act.GetAwaitableChoice())
+	}
+}
+
+func withAwaitableChoice(ctx workflow.Context, starter func(workflow.Context) workflow.Future,
+	awaitChoice *kitchensink.AwaitableChoice) error {
```

Comment:
```suggestion
func withAwaitableChoice(
	ctx workflow.Context,
	starter func(workflow.Context) workflow.Future,
	awaitChoice *kitchensink.AwaitableChoice,
) error {
```

A bit more idiomatic to do it like this when spanning multiple lines in a signature